### PR TITLE
Added dutch translations for keycloak 12

### PR DESCRIPTION
**Pull request by antek-drzewiecki**
Since Redhat SSO is not properly working. I was not able to see the JIRA board and find a ticket for this issue.

This pull requests adds translations for the community edition of keycloak. All except for WebAuthn since I can't reproduce the flows.

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties [hunk 1]
```diff
@@ -12,11 +12,17 @@ doDecline=Afwijzen
 doForgotPassword=Wachtwoord vergeten?
 doClickHere=Klik hier
 doImpersonate=Identiteit overnemen
+doTryAgain=Probeer het opnieuw
+doTryAnotherWay=Probeer het op een andere manier
+doConfirmDelete=Bevestig verwijderen
+errorDeletingAccount=Een fout is opgetreden bij het verwijderen van de account
+deletingAccountForbidden=U heeft onvoldoende permissies om de account te verwijderen
```
Comment (fndejong) [inline themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties]: het account

File: themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties [hunk 2]
```diff
@@ -292,3 +323,38 @@ console-verify-email=U bent verplicht om uw e-mailadres te verifiëren. Een e-ma
...
+
+# Openshift messages
+openshift.scope.user_info=Gebruikersinformatie
+openshift.scope.user_check-access=Gebruikers toegang informatie
+openshift.scope.user_full=Volledige toegang
+openshift.scope.list-projects=Lijst van projecten
+
+# SAML authentication
+saml.post-form.title=Authenticatie verwijzing
+saml.post-form.message=U wordt verwezen, een moment geduld alstublief.
```
Comment (fndejong) [inline themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties]: doorverwezen?

alstublieft*

File: themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties [hunk 3]
```diff
@@ -292,3 +323,38 @@ console-verify-email=U bent verplicht om uw e-mailadres te verifiëren. Een e-ma
 console-email-code=E-mail Code:
 console-accept-terms=Accepteert u de voorwaarden? [y/n]:
 console-accept=y
+
+# Openshift messages
+openshift.scope.user_info=Gebruikersinformatie
+openshift.scope.user_check-access=Gebruikers toegang informatie
```
Comment (fndejong) [inline themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties]: in theorie zou dit 1 woord zijn

**Review by fndejong — CHANGES_REQUESTED** [review]

File: themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties [hunk 4]
```diff
@@ -246,10 +275,12 @@ invalidAccessCodeMessage=Ongeldige toegangscode.
 sessionNotActiveMessage=Sessie inactief.
 invalidCodeMessage=Er is een fout opgetreden, probeer nogmaals in te loggen.
 identityProviderUnexpectedErrorMessage=Onverwachte fout tijdens de authenticatie met de identity provider
+identityProviderMissingStateMessage=Ontbrekende state parameter in het respons van de identity provider.
 identityProviderNotFoundMessage=Geen identity provider gevonden met deze naam.
 identityProviderLinkSuccess=Uw account is met succes gekoppeld aan {0} account {1}.
 staleCodeMessage=Deze pagina is verlopen. Keer terug naar uw applicatie om opnieuw in te loggen.
 realmSupportsNoCredentialsMessage=Realm ondersteunt geen enkel soort aanmeldgegeven.
+credentialSetupRequired=Kan niet inloggen, aanmeldgegeven moeten worden geconfigureerd.
```
Comment (TriMM-Berend) [inline themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties]: Aanmeldgegevens*

File: themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties [hunk 5]
```diff
@@ -82,24 +95,30 @@ offlineAccessScopeConsentText=Offline toegang
 samlRoleListScopeConsentText=Mijn rollen
 rolesScopeConsentText=Gebruikersrollen
 
+restartLoginTooltip=Opnieuw inloggen
+
 loginTotpIntro=U bent verplicht om tweefactor-authenticatie in te stellen om dit account te kunnen gebruiken
 loginTotpStep1=Installeer een van de volgende applicaties op uw mobile telefoon
```
Comment (fndejong) [inline themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:101]: mobiele* telefoon (zie ik net tijdens de demo)

File: themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties [hunk 6]
```diff
@@ -82,24 +95,30 @@ offlineAccessScopeConsentText=Offline toegang
...
 loginTotpAlgorithm=Algoritme
 loginTotpDigits=Cijfers
 loginTotpInterval=Interval
 loginTotpCounter=Teller
+loginTotpDeviceName=Toestelnaam
 
 loginTotp.totp=Time-based
 loginTotp.hotp=Counter-based
 
+loginChooseAuthenticator=Selecteer een login methode
```
Comment (fndejong) [inline themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:121]: inlogmethode (denk ik)

File: themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties [hunk 7]
```diff
@@ -148,12 +167,18 @@ role_manage-account-links=Beheer accountlinks
...
+client_account-console=Account Console
 client_security-admin-console=Security Admin Console
 client_admin-cli=Admin CLI
 client_realm-management=Realm-beheer
 client_broker=Broker
 
+requiredFields=Vereiste velden
+
 invalidUserMessage=Ongeldige gebruikersnaam of wachtwoord.
+invalidUsernameMessage=Ongeldig gebruikersnaam.
```
Comment (fndejong) [inline themes/src/main/resources-community/theme/base/login/messages/messages_nl.properties:179]: Ongeldige*

**Pull request closed without merging**